packages/main/src/themes/BusyIndicator.css

```diff
@@ -0,0 +1,94 @@
+:host(ui5-busyindicator) span[data-sap-ui-wc-root] {
```

Comment:
did you check if "hidden" hides the busy indicator

Reply:
Yes, it does